README.md

```diff
@@ -18,12 +18,12 @@ We will be building this app using Android Studio. This app will also consume th
  * Step 1 - Launch Android Studio -> New Project -> Create a new Android Project. Name your project AndroidPhotoApp.
 
 
-![minio_ANDROID2](https://github.com/minio/android-photo-app/blob/master/docs/screenshots/minio-ANDROID2.jpg?raw=true)
+![minio_ANDROID2](docs/screenshots/minio-ANDROID2.jpg?raw=true)
 
 
  * Step 2 - Select Phone & Tablet. In this example we choose the latest stable Marshmallow Android 6.0 SDK to compile and build this app. 
  
-![minio_ANDROID3](https://github.com/minio/android-photo-app/blob/master/docs/screenshots/minio-ANDROID3.jpg?raw=true)
+![minio_ANDROID3](docs/screenshots/minio-Android-os.png?raw=true)
```

Comment:
Same as above. Open the image in github. Right click and copy image address to get full path.